pkg/lsp/service.go

```diff
@@ -223,17 +223,11 @@ func (s *ServiceImpl) GetDocumentOutline(ctx context.Context, file model.File) (
 		return DocumentOutline{}, fmt.Errorf("project not found in context")
 	}
 
-	clients := s.getClients(ctx)
```

Comment:
This code block is no used. Probably was my initial take, where I later found a proper way of getting a client for the language.